src/main/java/me/earth/earthhack/impl/core/mixins/entity/living/MixinEntityLivingBase.java

```diff
@@ -100,10 +114,6 @@ public abstract class MixinEntityLivingBase extends MixinEntity
     @Shadow
     public abstract boolean isServerWorld();
 
-    @Override
-    @Invoker(value = "getArmSwingAnimationEnd")
-    public abstract int armSwingAnimationEnd();
```

Comment:
```java
@Override
@Invoker(/*value = "getArmSwingAnimationEnd"*/)
public abstract int invokeGetArmSwingAnimationEnd();
```
This should work to get around the mapping conflict, please test it first

Reply:
Does not work sadly. Still says mapping conflict for @ Invoker target.

Reply:
Did you make sure to comment out the parameters of the Invoker annotation?

Reply:
yes

Reply:
And you did rename `armSwingAnimationEnd` to `invokeGetArmSwingAnimationEnd`??? Cause it does not happen for me.

Reply:
Maybe I was too fast yesterday when testing. Will try when I get home.

Reply:
I cant get it work for me D:
I cant see where I would have done anything wrong.

Reply:
> 

Scrap the entire thing and do it again without looking at the old one